README.md

```diff
@@ -23,15 +23,15 @@
   </a>
 </p>
 
-Turbo is an next-generation toolchain for frontend development, written in Rust. It consists of 3 major parts:
+Turbo is a next-generation, Blazing Fast™, toolchain for frontend development, written in Rust. It consists of 3 major parts:
```

Comment:
I believe it should be:
```suggestion
Turbo is a next-generation, **Blazingly Fast™**, toolchain for frontend development, written in Rust. It consists of 3 major parts:
```

Reply:
This was discussed here https://github.com/vercel/turbo/pull/2414#pullrequestreview-1158966517

Reply:
@omar2205 that was taken offline

Reply:
Yeah, but I guess they settled on Blazing instead of Blazingly